code/game/objects/items/weapons/tools/simple_weapons.dm

```diff
@@ -53,6 +53,32 @@
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	tool_qualities = list(QUALITY_CUTTING = 20)
 
+/obj/item/tool/makeshiftaxe
+	name = "makeshift axe"
+	desc = "A heavy plasteel blade affixed to a welded metal shaft, for close up carnage"
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "makeshift_axe"
+	item_state = "makeshift_axe"
+	wielded_icon = "makeshift_axe_wielded"
+	matter = list(MATERIAL_STEEL = 2, MATERIAL_PLASTEEL = 1)
```

Comment:
you should make it recycle to 3 steel and plasteel too!

Reply:
> you should make it recycle to 3 steel and plasteel too!

fixed